protostar-rust/tests/e2e/running.rs

```diff
@@ -73,3 +74,29 @@ fn run_print_test() {
             Tests: 1 passed, 0 failed
         "#});
 }
+
+#[test]
+fn run_panic_decoding_test() {
+    let temp = assert_fs::TempDir::new().unwrap();
+    temp.copy_from("tests/data/panic_decoding_test", &["**/*"])
+        .unwrap();
+
+    let snapbox = runner();
+
+    snapbox
+        .current_dir(&temp)
+        .assert()
+        .success()
+        .stdout_matches(indoc! {r#"Collected 1 test(s) and 2 test file(s)
+            Running 0 test(s) from src/lib.cairo
+            Running 1 test(s) from tests/test_panic_decoding.cairo
+            [FAIL] test_panic_decoding::test_panic_decoding::test_panic_decoding
+            original value: [123], converted to a string: [{]
+            original value: [6381921], converted to a string: [aaa]
+            original value: [3618502788666131213697322783095070105623107215331596699973092056135872020480]
+            original value: [152]
+            original value: [124], converted to a string: [|]
+            original value: [149]
```

Comment:
Can we change it so we have the "trace" so the error message is visually separated? 

So it's like this:

```
Running 0 test(s) from src/lib.cairo
Running 1 test(s) from tests/test_panic_decoding.cairo

[FAIL] test_panic_decoding::test_panic_decoding::test_panic_decoding

Failure data:
    original value: [123], converted to a string: [{]
    original value: [6381921], converted to a string: [aaa]
    original value: 
    [3618502788666131213697322783095070105623107215331596699973092056135872020480]
    original value: [152]
    original value: [124], converted to a string: [|]
    original value: [149]

Tests: 0 passed, 1 failed
```

Also I'd add extend this test case with couple passing tests after and before this so we can see how this behaves in context of running multiple tests.

Reply:
Good idea